qutebrowser/misc/utilcmds.py

```diff
@@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
     cmd = runners.last_command[mode_manager.mode]
     commandrunner = runners.CommandRunner(win_id)
     commandrunner.run(cmd[0], count if count is not None else cmd[1])
+
+@cmdutils.register(maxsplit=1, no_cmd_split=True, name='debug-log-capacity')
```

Comment:
No need for `maxsplit=1` and `no_cmd_split=True` here (which control how arguments are splitted), but instead you should add `debug=True` so this command is only shown in the completion with `--debug`.